configuration/concepts/operative_progress.csv

```diff
@@ -0,0 +1,21 @@
+Uuid,Void/Retire,Fully specified name:en,Short name:en,Fully specified name:es,Short name:es,Description:en,Data class,Data type,Answers,Members,Units
+a957ff9c-80f1-4f31-8937-3bef0cfb8c0e,,Pre-Op Diagnosis,,Diagnóstico preoperatorio,Diagnóstico preoperatorio,,Misc,Text,,,
+f348e4b5-4124-4169-966a-2dff2a944566,,post-Op Diagnosis,,diagnóstico postoperatorio,diagnóstico postoperatorio,,Misc,Text,,,
+27fe6714-0bc6-4435-adb0-818538abe42c,,Procedure,,Procedimiento,Procedimiento,,Misc,Text,,,
+acda413c-56cc-48ab-825d-af58cb947f15,,Surgeon/ Assistant,,Cirujano/ Asistente,Cirujano/ Asistente,,Misc,Text,,,
+3ca389b4-6b9d-487c-bdd9-73d2cfeeab0c,,General,Gen,General,General,,Misc,N/A,,,
+5392d074-eed3-4f88-a8e9-5bae333e7589,,Monitired Anesthesia Care,Mac,Atención anestésica supervisada,Atención anestésica supervisada,,Misc,N/A,,,
+84ba7929-3714-4023-843a-2b1814563207,,Knee Block,,bloque de rodilla,bloque de rodilla,,Misc,N/A,,,
+7dd25939-b4e5-4657-8c69-ac4993f183b5,,Axillary Block,Ax Block,Bloque Axilar,Bloque Axilar,,Misc,N/A,,,
+0f087406-b092-419c-a9ae-bd32c9e29a43,,Interscalene Block,,Bloque interescalénico,Bloque interescalénico,,Misc,N/A,,,
+20fafddb-bfd4-421e-acfb-639454741e94,,Intravenous Block,IC Block,Bloque intravenoso,Bloque intravenoso,,Misc,N/A,,,
+3dbb2d82-ee96-43a5-a905-0401c2d8b1a9,,Local anesthesia and Sedative,Loc//Sed,Anestesia local y Sedante,Ubicación//Sed,,Misc,N/A,,,
+ea6ac7de-d7cf-4105-aadb-b1e489536a7e,,Local,Local,Local,Local,,Misc,N/A,,,
+3017f680-5c3c-4f7b-94e1-6afda9e07eaf,,Anesthesia Type,Anesthesia Type,Tipo de anestesia,Tipo de anestesia,,ConvSet,N/A,,Gen; Mac; Local; Knee Block; Ax Block; Interscalene Block; IV Block; Loc//Sed,
+b5ab7bc0-bffb-4dc8-b832-2e1713f06944,,Estimated Blood Loss,EBL,Pérdida de sangre estimada,Pérdida de sangre estimada,,Misc,Numeric,,,mls
+64fe56e6-5b52-4cdb-adc2-b6d50bfdee63,,Complications,,Complicaciones,Complicaciones,,Misc,Text,,,
+08f1f638-b3c7-4040-8ae9-c95a5df59755,,Specimens,,especímenes,especímenes,,Misc,Text,,,
+59185273-c5b8-4e73-8ba2-c68710f6c507,,Drains,,Drenajes,Drenajes,,Misc,Text,,,
+a1da87f9-c4af-4598-accd-489268a2d325,,Condition upon Discharge from OR,,Condición al alta del quirófano,Condición al alta del quirófano,,Misc,Text,,,
+0934658e-fee3-44e3-a8f2-292dc3587f54,,Time of Procedure,,Tiempo de Procedimiento,Tiempo de Procedimiento,,Misc,Text,,,
+2554a66d-46fe-4f5f-813a-2b491716bddf,,Post Operative Instruction,,Instrucción postoperatoria,Instrucción postoperatoria,,Misc,Text,,,
```

Comment:
Lets use the FSN as 'Post-Operative Instruction'

Reply:
Done